tools/run_tests/task_runner.py

```diff
@@ -73,6 +73,18 @@ def _create_build_map():
                   default='report_taskrunner_sponge_log.xml',
                   type=str,
                   help='Filename for the JUnit-compatible XML report')
+argp.add_argument('--dry_run',
+                  default=False,
+                  action='store_const',
+                  const=True,
+                  help='Only print what would be run.')
+argp.add_argument(
+    '--inner_jobs',
+    default=None,
+    type=int,
+    help=
+    'Number of parallel jobs to use by each target. Passed as build_jobspec(inner_jobs=N) to each target.'
```

Comment:
Maybe we can add a short description for what builds support inner_job?

Reply:
I added comments into the build_jobspec() functions, I think having a list of what's supported in the arg helptext would be hard to maintain. Also, as a followup I plan to provide inner_jobs support for all the tasks where it makes sense.